.github/labeler.yml

```diff
@@ -2,20 +2,33 @@
 Multicast:
   - changed-files:
     - any-glob-to-any-file:
-      - setup.py
+      - 'setup.*'
       - multicast/*.py
       - LICENSE
       - LICENSE.md
+      - pyproject.toml
 
 # Add 'Lint' label to any lint file changes
 Linter:
   - changed-files:
     - any-glob-to-any-file:
-      - .hound.yml
```

Comment:
As part of #55 we are reconsidering use of HoundCI - can re-add in future if actually needed.